zstacklib/zstacklib/utils/objgraph.py

```
@@ -0,0 +1,960 @@
+
+
+def _quote(s):
+    return (s.replace("\\", "\\\\")
+            .replace("\"", "\\\"")
+            .replace("\n", "\\n")
+            .replace("\0", "\\\\0"))
+
+
+def _get_obj_type(obj):
+    objtype = type(obj)
+    if type(obj) == InstanceType:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)